Sources/XCBuildSupport/XcodeBuildSystem.swift

```diff
@@ -226,7 +231,7 @@ struct XCBBuildParameters: Encodable {
     }
 
     struct SettingsOverride: Encodable {
-        var commandLine: XCBSettingsTable? = nil
+        var synthesized: XCBSettingsTable? = nil
```

Comment:
@jakepetroules this should be more future proof, right?

Reply:
Yes, thanks!